### DISCUSSION TITLE
How to prevent library shapes from breaking when resizing them?

### DISCUSSION THREAD

**Discussion by fedme** [excalidraw/excalidraw#5490] [Q&A] [answered]
Hi!
I am using a few shapes/drawings from the free library but my experience with most of them is that they break when resizing them.
Am I doing something wrong?

Example:
![image](https://user-images.githubusercontent.com/3295133/181081293-4360dde3-1a95-4212-90d5-98db326b9021.png)

**Comment by alex-kim-dev** [comment, marked as answer]
This was recently fixed in #5560 

![Peek 2022-08-15 12-12](https://user-images.githubusercontent.com/45559664/184592164-74bc9447-f1ec-4bd9-8745-a5d7d5b2c4a3.gif)
